code/game/objects/items/vending_items.dm

```diff
@@ -37,8 +37,8 @@
 /obj/item/vending_refill/boozeomat
 	machine_name = "Booze-O-Mat"
 	icon_state = "refill_booze"
-	charges = list(54, 4, 0)//of 159 standard, 12 contraband
-	init_charges = list(54, 4, 0)
+	charges = list(60, 4, 0)//of 168 standard, 12 contraband

```

Comment:
This number is wrong.
Should be 58, to refill for a total of 174.

Reply:
I counter, there are 180 total items in total